pyproject.toml

```diff
@@ -15,7 +15,7 @@ pandas = "^1.4.2"
 pyhumps = "^3.7.2"
 json-logging = "^1.3.0"
 tomlkit = "^0.11.4"
-microdata-tools = "^0.4.0"
+microdata-tools = "^0.6.0"
```

Comment:
microdata-validator dependency is now not needed?

Reply:
It should be removed, I can see some other refs to "micordata-tools" (readme, doc-string) so I will update those too. Good catch!